### ISSUE TITLE
Github crashes and won't restart

### ISSUE THREAD

**Issue by donbode** [desktop/desktop#11776]
17 min after a successful commit and push, a similar commit and push failed.  I got the following error message
![image](https://user-images.githubusercontent.com/57908896/110819710-ac4fc880-824b-11eb-9ddb-b0ff5fc7a0dd.png)
![image](https://user-images.githubusercontent.com/57908896/110822467-53cdfa80-824e-11eb-8f53-a38f3a6933bf.png)
I sought that address and all I could find was
C:\Users\boded\AppData\Local\GitHubDesktop\app-2.6.6\resources\app 
and at that location was
![image](https://user-images.githubusercontent.com/57908896/110820008-f0db6400-824b-11eb-9009-6d82ab3c3fd9.png)

Now when I try to open GitHub, the cursor remains visible for a couple of seconds and then disappears.  GitHub does not start.
I find no remnants in Task Manager.

I am running Win10 and have Norton 360 antivirus.  

If I go to Git in Visual Studio, I am able to "commit and push".  I know this by going to    
C:\Users\b****\Documents_G\GitHub\*********** where I find the expected result using "git log --oneline."

I am an amateur and 79 years old, so forgive me if I don't speak the lingo well.  I erroneously began this inquiry on a different thread because that problem sounded like mine.  The expert kindly suggested that the problem might be the antivirus.  Maybe it is the antivirus, but I am sceptical since find nothing in the history saying that action was taken on these files  Granted, I could have missed it, but also I have been using this setup for a year with no problem  Additionally, Visual Studio's Git works.

My immediate concern is how to recover GitHub.  Must I re-install?  What do I do?  Should I tell Noron to not remove certain files and if so, what are they?

Thank you for your patience.
Don

**Comment by sergiou87** [comment]
Hello again @donbode! 👋 

Thank you for opening a new issue ❤️  , the other one seemed similar but was actually different.

Aside from Norton, do you also have Windows Defender? I've seen Windows Defender once "stealing" the executables from me and causing issues like this 😂  If Norton doesn't show anything in its History screen, maybe [Windows Defender does](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-antivirus/restore-quarantined-files-microsoft-defender-antivirus#:~:text=If%20you%20are%20certain%20a,an%20action%2C%20such%20as%20restore.)?

Regardless of what you do with your antivirus, you will probably need to reinstall GitHub Desktop and it should Just Work™ after that. If the problem happens again, my advice is that you disable **temporarily** all your antivirus systems, reinstall GitHub Desktop, and give it another try for a few hours.

If after a few hours with the antivirus disabled, GitHub Desktop keeps working, we can probably assure the problem is in either Norton or Windows Defender. However, **beware:** Windows Defender re-enables itself automatically after X minutes/hours.

I hope that helps! Please, give all that a try and let me know if it still happens!

**Comment by donbode** [comment]
I checked in Task Manager and there is a single entry:  Windows Defender
SmartScdreen. I looked at the site Windows Security that you suggested and
it led to a report on my computer "Windows Defender Antivirus turned off".
Perhaps it is off since I could not find any quarantine items, actions
blocked, or security problems anywhere.

That led me to re-examine Norton 360 for quarantined items.  I found two
adjacent entries in Norton that may be what you are citing as a cause.
They report "Action Blocked"  and say  [image: image.png]
I could not find further info.  The only glitch in this argument is that
all of the information relates to the .exe I am creating and not Visual
Studio or GitHub.

I'll reinstall and report if I have further problems while I hope for the
best
Don
